packages/iris-grid/src/IrisGridTheme.ts

```diff
@@ -49,13 +49,60 @@ export type IrisGridThemeType = GridThemeType & {
   iconSize: number;
 };
 
+export interface IrisGridDensity {
+  cellHorizontalPadding: number;
+  headerHorizontalPadding: number;
+  minColumnWidth: number;
+  rowHeight: number;
+  font: string;
+  headerFont: string;
+  iconSize: number;
+  columnHeaderHeight: number;
+  filterBarHeight: number;
+}
+
+export const NORMAL_DENSITY_THEME = {
```

Comment:
We may want to remove these from being defined in the theme created below and just spread `NORMAL_DENSITY_THEME` in there by default. Otherwise we're re-defining these values in two places, if we ever do bother to change one of them might forget to change the other...

That also being said - we should use `IrisGridTheme['header-font']` for `headerFont` here, and all the other things from `IrisGridTheme.module.scss` instead of redefining here.